DataConnectors/GoogleWorkspaceReports/GWorkspaceReportsAPISentinelConnector/__init__.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import print_function
+import pickle
+from googleapiclient.discovery import build
+import datetime
+import json
+import base64
+import hashlib
+import hmac
+import requests
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+import azure.functions as func
+import logging
+import os
+import time
+
+customer_id = os.environ['WorkspaceID'] 
+shared_key = os.environ['WorkspaceKey']
+pickle_str = os.environ['GooglePickleString']
+pickle_string = base64.b64decode(pickle_str)

```

Comment:
Yes, the output is base64 encoded. But to make the main script work correct we need the original pickle value. So here we’re doing decode of it.
This way is used because we can’t use original pickle (non-base64) value, it contains special characters that lead to error while saving the env variable.